### ISSUE TITLE
When a attribute aria-expanded is false, rendering is failed (dioxus-web 0.3.0)

### ISSUE THREAD

**Issue by chengchangwu** [DioxusLabs/dioxus#776]
**Problem**

When a attribute aria-expanded is false, rendering is failed.
This issue is different from the #760.

**Steps To Reproduce**

- Run the code snippet

```
#![allow(non_snake_case)]
// import the prelude to get access to the `rsx!` macro and the `Scope` and `Element` types
use dioxus::prelude::*;

fn main() {
    // launch the web app
    dioxus_web::launch(App);
}

// create a component that renders a div with the text "Hello, world!"
fn App(cx: Scope) -> Element {
    let aria_expanded = false;
    cx.render(rsx! {
        div {
            button {
                aria_expanded: aria_expanded,
                "Hello, world!"
            }
        }
    })
}
```

- DevTool console outputs logs:

```
Uncaught (in promise) ReferenceError: bool_attrs is not defined
    at SetAttributeInner (inline0.js:82:46)
    at run (inline0.js:126:2957)
    at dioxus_interpreter_js::sledgehammer_bindings::run::hbc946e59c0d2e7a2 (dioxus_bg.wasm:0x1fbfc8)
    at dioxus_interpreter_js::sledgehammer_bindings::Channel::flush::ha7876ac270261519 (dioxus_bg.wasm:0xbc7bf)
```

**Expected behavior**

A button is rendered.

**Environment:**
 - Dioxus version: [v0.3.1]
 - Rust version: [1.66.1]
 - OS info: [Ubuntu 20.04]
 - App platform: [web]

**Questionnaire**

- [x] I'm interested in fixing this myself but don't know where to start
- [ ] I would like to fix and I have a solution
- [ ] I don't have time to fix this right now, but maybe later

**Comment by Demonthos** [comment]
This works on the master branch. My guess is this was fixed by https://github.com/DioxusLabs/dioxus/pull/761
